src/features/birthdayNotifications/pages/RecreditBirthdayNotification.native.test.tsx

```diff
@@ -12,7 +12,7 @@ import { RecreditBirthdayNotification } from './RecreditBirthdayNotification'
 
 jest.mock('libs/firebase/analytics/analytics')
 jest.mock('features/auth/context/AuthContext')
-jest.mock('features/profile/api/usePatchProfile', () => ({
+jest.mock('/queries/profile/usePatchProfileMutation', () => ({
```

Comment:
je crois que je n'ai pas encore vu de mock commencer par un /

```suggestion
jest.mock('queries/profile/usePatchProfileMutation', () => ({
```

Reply:
il y en a d'autre dans d'autres fichiers